Makefile

```diff
@@ -152,7 +153,7 @@ PROTOC_VER ?= 23.4
 PROTOC_GEN_GO_VER ?= 1.31.0
 PROTOC_GEN_GO_GRPC_VER ?= 1.3.0
 BUF_VERSION ?= 1.23.1
-
+CONTROLLER_GEN_VERSION ?= v0.11.1
```

Comment:
Last version is v0.13.0. Do we want to upgrade?

Reply:
upgraded